js/main.js

```diff
@@ -186,7 +187,21 @@ var loadProfile = function(landingRoute) {
 
 $(document).ajaxError(function(event, jqxhr, settings, thrownError) {
   if (jqxhr.status === 401) {
-    launchServerConnect();
```

Comment:
what's happening here is that when the session times-out, the next API call will return a 401 - unauthorized. Previously, we were opening the Server Connect modal, which upon re-auth would re-start the app.

Now, rather than kicking off the modal, we try and authenticate, and if we can, we just refresh the route. There will still be a flicker as the route refreshes, including a potential Message Modal from the fail handler in the page's view, but it's less heavy-handed than bring up the Server Connect modal and subsequently re-starting the whole app.

Also, @cpacia is looking into increasing the session timeout threshold, so this scenario will not occur as frequently.